---a/net/minecraft/entity/ai/EntityAILookAtTradePlayer.java
+++b/net/minecraft/entity/ai/EntityAILookAtTradePlayer.java
@@ -3,28 +3,23 @@
 import net.minecraft.entity.passive.EntityVillager;
 import net.minecraft.entity.player.EntityPlayer;
 
-public class EntityAILookAtTradePlayer extends EntityAIWatchClosest
-{
+public class EntityAILookAtTradePlayer extends EntityAIWatchClosest {
+
     private final EntityVillager theMerchant;
     private static final String __OBFID = "CL_00001593";
 
-    public EntityAILookAtTradePlayer(EntityVillager p_i1633_1_)
-    {
+    public EntityAILookAtTradePlayer(EntityVillager p_i1633_1_) {
         super(p_i1633_1_, EntityPlayer.class, 8.0F);
         this.theMerchant = p_i1633_1_;
     }
 
-    public boolean shouldExecute()
-    {
-        if (this.theMerchant.isTrading())
-        {
+    public boolean shouldExecute() {
+        if (this.theMerchant.isTrading()) {
             this.closestEntity = this.theMerchant.getCustomer();
             return true;
-        }
-        else
-        {
+        } else {
             return false;
         }
     }
-}
+}
